byzcoin/service.go

```diff
@@ -186,13 +180,17 @@ func (s *Service) CreateGenesisBlock(req *CreateGenesisBlock) (
 		return nil, err
 	}
 
+	// This is the nonce for the trie.
+	nonce := GenNonce()
```

Comment:
i've changed it to private, it won't be needed anyway after https://github.com/dedis/cothority/issues/1442 is done because it won't use a nonce anymore

Reply:
actually, I had to bring this back because the simulation test fails if we don't use real nonces. but anyway, the `GenNonce` will go away when we have proper client transactions